Formula/srt.rb

```diff
@@ -0,0 +1,28 @@
+class Srt < Formula
+  desc "Secure, Reliable, Transport"
+  homepage "http://srtalliance.org"
+  url "https://github.com/Haivision/srt/archive/v1.2.0.tar.gz"
+  sha256 "ca22506cfafa18fb1f9c6dd5ceebe5e56c1b205105b2dd3736701aeea3f4145a"
+
+  head do
```

Comment:
Remove the `head do` block, just add `head "https://whatever"` after the `sha256` line (no empty line in-between)